frontend/controller/serviceworkers/cache.js

```diff
@@ -0,0 +1,131 @@
+import sbp from '@sbp/sbp'
+import { NOTIFICATION_TYPE } from '~/shared/pubsub.js'
+
+const CACHE_VERSION = '1.0.0'
+const CURRENT_CACHES = {
+  assets: `assets-cache_v${CACHE_VERSION}`
+}
```

Comment:
Should this version number be tied to some environment variable, e.g. `GI_VERSION` or something else?

Reply:
I was thinking about that. It could, but I don't think it needs to, so long as it's updated if breaking changes to the cache are made.